README.md

```diff
@@ -58,6 +58,79 @@ The networks object includes a set of networks for which JWTs will be verified o
 
 Look in [uport-lite](https://github.com/uport-project/uport-lite) for the default networks and registries which will be queried for JWT verification.
 
+## Authentication
```

Comment:
I think we should probably change the wording here to be Enhanced  or Strong Authentication as regular Uport Connect also is Authentication.